pkg/grpc/requestinfo/requestinfo.go

```diff
@@ -76,6 +77,11 @@ func (s Source) GetSourceIP() string {
 	)
 }
 
+// HeaderGetter interface abstracts http.Header and metadata.MD types.
```

Comment:
Well, that's not true: `func (h Header) Get(key string) string` : ). I'd call it something like `HeadersMultiMap`.